cmd/lncli/commands.go

```diff
@@ -1733,19 +1733,7 @@ func retrieveFeeLimit(ctx *cli.Context) (*lnrpc.FeeLimit, error) {
 	return nil, nil
 }
 
-func confirmPayReq(ctx *cli.Context) error {
-	args := ctx.Args()
-
-	var payReq string
-	switch {
-	case ctx.IsSet("pay_req"):
-		payReq = ctx.String("pay_req")
-	case args.Present():
-		payReq = args.First()
-	default:
-		return fmt.Errorf("pay_req argument missing")
-	}
-
+func confirmPayReq(ctx *cli.Context, payReq string) error {
```

Comment:
We can actually pass in the `lnrpc.LightningClient` rather than the `cli.Context` like we do in `executeChannelClose` :)

Reply:
Sure, I did the change, but it means that the client creation is duplicated at both function calls.

Reply:
Yep, so then we can also just do the same thing with `sendPaymentRequest` and then we can have the client definition at the top of `payInvoice` like all the other commands.

Reply:
Sounds good, done.